Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM registry.access.redhat.com/rhscl/ruby-23-rhel7
+
+# sshd
+USER root
+RUN yum install -y --setopt=tsflags=nodocs openssh-server libicu-devel && \
```

Comment:
There are 5 RUN commands in sequence here which will create 5 separate image layers. I tried briefly to make them one, but I didn't succeed. They'd need to use the form in L14 that has `["bash", "-c", "..."], since otherwise the SCL doesn't seem to get enabled. I might come back to it at some point.